force-app/main/default/classes/ContactTriggerProfitService.cls

```diff
@@ -0,0 +1,25 @@
+public with sharing class ContactTriggerProfitService {
```

Comment:
Te same komentarze tutaj co do ExpenseServicu, ale to możesz nawet zakomentować i odpuśćić liczenie profitu sobie póki co, co by nie komplikować